centreon/www/front_src/src/Header/Poller/PollerSubMenu/PollerSubMenu.tsx

```diff
@@ -0,0 +1,102 @@
+import { makeStyles } from 'tss-react/mui';
+import { isEmpty } from 'ramda';
+
+import { Typography, Button, List, ListItem } from '@mui/material';
+
+import ExportConfiguration from './ExportConfiguration';
+
+const useStyles = makeStyles()((theme) => ({
+  link: {
+    textDecoration: 'none'
+  },
+  list: {
+    padding: 0
+  },
+  listItem: {
+    '&:not(:last-child)': {
+      borderBottom: `1px solid ${theme.palette.divider}`
+    },
```

Comment:
As you are using MUI Lists, you get rid of this
```suggestion
```

Reply:
no there is no divider by default when using the MUI list, 
I can add the `divider` props in the items but it adds it to the last one as well. 
I could do something like `list.map((el, i) => <listItem divider={I !== list.length}) >` but it seems completely overkill to use javascript to handle something that css is perfectly good at doing...